app/components/Admin/ChangeRequest.js

```diff
@@ -16,23 +14,23 @@ class ChangeRequest extends React.Component {
 
   componentDidMount() {
     this.retrieveModifiedObject();
-    let fields = this.props.changeRequest.field_changes;
-    let tempChangeRequestFields = {};
-    fields.forEach(field => {
+    const fields = this.props.changeRequest.field_changes;
+    const tempChangeRequestFields = {};
+    fields.forEach((field) => {
       tempChangeRequestFields[field.field_name] = field.field_value;
     });
 
     this.setState({ changeRequestFields: tempChangeRequestFields });
-
   }
 
   retrieveModifiedObject() {
-    let changeRequest = this.props.changeRequest;
-    let resource = changeRequest.resource;
-    //"ChangeRequest" is 13 characters, so this will give us the first part of the string
-    let objectType = changeRequest.type;
+    const changeRequest = this.props.changeRequest;
+    const resource = changeRequest.resource;
+    // "ChangeRequest" is 13 characters, so this will give us the first part of the string
+    const objectType = changeRequest.type;
+    let resourceNotes;
```

Comment:
I don't have strong opinions about it. I'm guessing it doesn't like the variable being declared inside of the case statement because case statements don't provide scope, so the scope of the variable is actually greater than what it looks.

It looks like you ended up fixing it by making it a const anyway, so let's not change the linter.